_pytest/terminal.py

```diff
@@ -19,12 +19,39 @@
     EXIT_USAGEERROR, EXIT_NOTESTSCOLLECTED
 
 
+import argparse
+
+
+class MoreQuietAction(argparse.Action):
```

Comment:
Please add a docstring documenting why this class exists.

Reply:
I don't understand why we need this class. What does it fix?

Reply:
it implements a negative counter and backward compat hacking, it can be simplified later on but its necessary since argparse itself is kinda shitty ^^

Reply:
ah. It is needed to get rid of `config.option.verbose -= config.option.quiet`, right?

Reply:
yup